x-pack/plugins/search_inference_endpoints/common/types.ts

```diff
@@ -7,6 +7,7 @@
 
 export enum APIRoutes {
   GET_INFERENCE_ENDPOINTS = '/internal/inference_endpoints/endpoints',
+  DELETE_INFERENCE_ENDPOINT = '/internal/inference_endpoint/{type}/{id}',
```

Comment:
is this path going to limit our options in the future by not having another qualifier after `/internal/inference_endpoint/`? should this at least be:

```suggestion
  DELETE_INFERENCE_ENDPOINT = '/internal/inference_endpoint/endpoints/{type}/{id}',
```

Reply:
Updated.